### PR TITLE
OTBR service discovery codelab

### PR DESCRIPTION
**Pull request by j-noel-longhand**
A handful of clarifications and corrections

### CODE REVIEW

File: site/en/codelabs/openthread-border-router/index.lab.md [hunk 1]
```diff
@@ -216,21 +216,25 @@ Done
...
 
 > aside positive
 > 
-> **Note:** The `OT_SRP_CLIENT` build option enables the SRP client side implementation on the end device, while the option `OT_ECDSA` is required by `OT_SRP_CLIENT` to create signatures. Here only specifies the options required for SRP Client feature, please append the other options you need.
+> **Note:** The `OT_SRP_CLIENT` build option enables the SRP client side implementation on the end device, while the option `OT_ECDSA` is required by `OT_SRP_CLIENT` to create signatures.
+
+So the full build invocation should look like this:
+
+```console
+$ script/build nrf52840 USB_trans -DOT_SRP_CLIENT=ON -DOT_ECDSA=ON -DOT_JOINER=ON -DOT_THREAD_VERSION=1.2
```
Comment (wgtdkp) [inline site/en/codelabs/openthread-border-router/index.lab.md:236]:
```suggestion
$ script/build nrf52840 USB_trans -DOT_SRP_CLIENT=ON -DOT_ECDSA=ON -DOT_THREAD_VERSION=1.2
```
I don't think we need the joiner feature.

**Review by wgtdkp — APPROVED** [review]
LGTM 👍

**Review by Vyrastas — APPROVED** [review]
LGTM pending @wgtdkp's suggestion

File: site/en/codelabs/openthread-border-router/index.lab.md (hunk 1 above)
Comment (jwhui) [reply site/en/codelabs/openthread-border-router/index.lab.md:236]:
```suggestion
$ script/build nrf52840 USB_trans -DOT_SRP_CLIENT=ON -DOT_ECDSA=ON
```

I believe `OT_THREAD_VERSION` default is now 1.2, so we could probably remove that as well.

@wgtdkp , do you agree?

Comment (wgtdkp) [reply site/en/codelabs/openthread-border-router/index.lab.md:236]: yeah, agree!

File: site/en/codelabs/openthread-border-router/index.lab.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Reboot the Raspberry Pi to take the changes to effect.
 
 OTBR supports a 15.4 radio chip in  [Radio Co-Processor](https://openthread.io/platforms#radio-co-processor-rcp) (RCP) mode. In this mode, the OpenThread stack is running on the host side and transmits/receives frames over the IEEE802.15.4 transceiver.
 
-Follow  [step 4 of this codelab](https://openthread.io/codelabs/openthread-hardware#3) to build and flash a nRF52840 RCP device. You need an additional option `-DOT_THREAD_VERSION=1.2` for the build step:
+Follow  [step 4 of the *Build a Thread network with nRF52840 boards and OpenThread* codelab](https://openthread.io/codelabs/openthread-hardware#3) to build and flash a nRF52840 RCP device. You need an additional option `-DOT_THREAD_VERSION=1.2` for the build step:
 
 ```console
 $ script/build nrf52840 USB_trans -DOT_THREAD_VERSION=1.2
```
Comment (wgtdkp) [inline site/en/codelabs/openthread-border-router/index.lab.md:120]:
```suggestion
Follow  [step 4 of the *Build a Thread network with nRF52840 boards and OpenThread* codelab](https://openthread.io/codelabs/openthread-hardware#3) to build and flash a nRF52840 RCP device:

```console
$ script/build nrf52840 USB_trans
```

File: site/en/codelabs/openthread-border-router/index.lab.md (hunk 1 above)
Comment (jwhui) [inline site/en/codelabs/openthread-border-router/index.lab.md:236]:
```suggestion
$ script/build nrf52840 USB_trans -DOT_SRP_CLIENT=ON -DOT_ECDSA=ON
```

**Review by jwhui — APPROVED** [review]

**Pull request merged**